### ISSUE TITLE
Sandbox Errors

### ISSUE THREAD

**Issue by dzhuang** [inducer/relate#186]
It raises ScannerError: mapping values are not allowed here in "", line 2, column 11: id: eigvec ^

The testing code is from [sample](https://github.com/inducer/relate-sample/blob/master/flows/quiz-test.yml)
```
        type: TextQuestion
        id: eigvec
        value: 2
        prompt: |
            # What's an eigenvector?
            Yadda ___________________ yadda.
        answers:

          - <plain>matrix
          - <case_sens_plain>Eigenmatrix
          - <regex>(?:linear\s+)?\s*map
          - <case_sens_regex>(?:operator\s+)?\s*map

```

**Comment by inducer** [comment]
You need to remove the indentation.

**Comment by dzhuang** [comment]
I remembered originally it won't require to remove indentation. I think the previous way is better than what it is now. In the past, I just copied working snippet directly from sandbox to my .yml, without more editing need.

**Comment by inducer** [comment]
Can you cook up a patch?

Here's a piece that might be useful:
https://github.com/inducer/pytools/blob/master/pytools/py_codegen.py#L98

**Comment by dzhuang** [comment]
Thank you for the great suggestion!

**Comment by dzhuang** [comment]
Should be fixed by #187
